src/main/assembly/dist/cfg/application.properties

```
@@ -114,12 +116,13 @@ mail.bounceAddress=info@dans.knaw.nl
 mail.bccs=
 
 # directory with depositConfirmation.txt and  depositConfirmation.html
-mail.template=opt/dans.knaw.nl/easy-deposit-api/cfg/template
+mail.template=/opt/dans.knaw.nl/easy-deposit-api/cfg/template
 
-# maximum number of lines alias files in attached files.txt
-file.limit=500
+# maximum number of lines alias files in (currently unzipped) attached files.txt
+# arbitrary assumption of an average 2000 chars per files -> attachment of 20MB
+attached-file-list.limit=10000
 
 agreement-generator.url=http://localhost:20210/agreement
-agreement-generator.accept=text/html
+agreement-generator.accept=application.pdf
```

```suggestion
agreement-generator.accept=application/pdf
```
I guess you mean this?